rskj-core/src/main/java/org/ethereum/net/MessageQueue.java

```diff
@@ -103,10 +103,13 @@ public void sendMessage(Message msg) {
             hasPing = true;
         }
 
-        if (msg.getAnswerMessage() != null)
-            requestQueue.add(new MessageRoundtrip(msg));
-        else
-            respondQueue.add(new MessageRoundtrip(msg));
+        Queue<MessageRoundtrip> queue = msg.getAnswerMessage() != null ? requestQueue : respondQueue;
+        queue.add(new MessageRoundtrip(msg));
+
+        int queueSize = queue.size();
```

Comment:
This shouldn't be merged, it was for debugging something only

Reply:
Debugging info removed :+1: